lib/suspenders/app_builder.rb

```diff
@@ -1,6 +1,14 @@
+require "forwardable"
+
 module Suspenders
   class AppBuilder < Rails::AppBuilder
     include Suspenders::Actions
+    extend Forwardable
+
+    def_delegators :heroku_adapter,
```

Comment:
The thing is that `rails/delegation` (and ActiveSupport in general I guess) is not loaded at this point, at least from what I could gather from the tests. Do you feel it is worth loading it?